### PR TITLE
use line loader for ad

### PR DESCRIPTION
**Pull request by zhouyx**
part for #5918.
Create a new style loading indicator, and apply it to ad first. 

cc: @jasti

### CODE REVIEW

File: css/amp.css [hunk 1]
```diff
@@ -323,6 +323,37 @@ i-amp-scroll-container.amp-active {
   visibility: hidden;
 }
 
+.-amp-loader-line {
+  /* TODO(zhouyx): Confirm on final height, color */
+  position: absolute;
+  top:0;
+  left: 0;
+  right: 0;
+  height: 2px !important;
```
Comment (lannka) [inline css/amp.css]: can we avoid `!important`? probably with a more accurate selector: `div.-amp-loader-line`?

File: css/amp.css [hunk 2]
```diff
@@ -323,6 +323,37 @@ i-amp-scroll-container.amp-active {
...
+  height: 2px;
+  overflow: hidden !important;
+  background-color: #696969;
+  display: block;
+}
+
+.-amp-loader-moving-line {
+  display: block;
+  position: absolute;
+  left: -50%;
```
Comment (jridgewell) [inline css/amp.css]: Shouldn't this start at `-100%`?

Comment (zhouyx) [inline css/amp.css]: woops, changed.

**Comment by abeck000** [comment]
cc ux @edlea-g

**Pull request merged**